tests/FacebookDataDeletionBackfillCommandTest.php

```
@@ -0,0 +1,147 @@
+        unlink($csv_path);
+    }
+
+    public function testDoesNotLogExternalId(): void
+    {
+        Log::spy();
+        $asid = '555000444';
+        $csv_path = $this->writeCsvFixture([$asid]);
+
+        Artisan::call(self::Command, ['path' => $csv_path]);
+
+        Log::shouldNotHaveReceived('debug', function (string $message) use ($asid) {
```

PHPStan (2.2.7)

[error] Call to an undefined static method Illuminate\Support\Facades\Log::shouldNotHaveReceived().


(staticMethod.notFound)